mcl/kzg/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+max_width = 110
```

Comment:
Do we really need custom linter config? We prefer defaults mostly.

Reply:
Good catch, this didn't need to be committed. Note that the whole source tree is not rustfmt'ed, I just wanted to format single working file and I'm not a fan of aggressive wrapping. Removed.